### PR TITLE
Math-Magician Full-Webpage

### PR DESCRIPTION
**Pull request by jaamanu**
Style calculator
Add home page
Add navigation bars
Add quote page
Style homepage
Style navigation bar
style quotes page
Add content to pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+import Home from './components/home';
+import Navbar from './components/navigation';
+import Quotes from './components/quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
```
Comment (HAMMAS-SALEEM) [inline src/App.js:9]: image

- [ ] Your readme file you don't have a descriptive title. Kindly change it to **Math Magician** to make your readme file more descriptive :blush:

Comment (HAMMAS-SALEEM) [inline src/App.js:9]: image

- [ ] Your Built with section should have the technologies and languages which you have used in this project :+1: to make your readme more descriptive :+1:

Comment (HAMMAS-SALEEM) [inline src/App.js:9]: image

- [ ] This Getting Started section should have those commands which are mandatory to run to use your application :blush: to make your readme more descriptive :+1:

```
git clone git@github.com:jaamanu/maths-magician.git
npm install
npm start
```

Comment (HAMMAS-SALEEM) [inline src/App.js:9]: image

- [ ] You have added a link to your license file but this link is broken :fearful: I guess you forgot to add the license file :smile: No worries you can add the license file using these [guildines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @jaamanu :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitFlow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @jaamanu, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

Please do consider, as requested by the previews reviewer:
- [ ] Please kindly implement the changes as requested by the previous reviewer as the will make your readme file more informative and professional

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaamanu** [comment]
@Mihndim2020 thanks for the feedback, can you be specific about where the required changes I have not worked on by the first reviewer?

**Comment by arslanbisharat** [comment]
Hi @jaamanu,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**